### PR TITLE
Desktop version portfolio

### PR DESCRIPTION
**Pull request by ehmaddd**
### DESKTOP VERSION PORTFOLIO

The previous portfolio is now extended for the desktop too. Thanks to media queries, only using them was possible to create this webpage. For alignment, CSS features like Grid and Flex were very helpful. I hope you will find it good and enjoy reviewing it.

Regards

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,16 +28,16 @@
...
-                <li><a href=#"><img src="./images/github.png"></a></li>
-                <li><a href=#"><img src="./images/V.png"></a></li>
+                <li id="twitter"><a href=#"><img src="./images/twitter.png"></a></li>
+                <li id="linkedin"><a href=#"><img src="./images/in.png"></a></li>
+                <li id="m"><a href=#"><img src="./images/M.png"></a></li>
+                <li id="github"><a href=#"><img src="./images/github.png"></a></li>
+                <li id="v"><a href=#"><img src="./images/V.png"></a></li>
             </ul>
             <i class="fa-solid fa-chevron-down"></i>
         </section>
```
Comment (bobb-Rob) [inline index.html:43]: - [x] [REQUIRED] The goal of the project milestone is to make your portfolio website **_RESPONSIVE_**, meaning that 
- Desktop view starts from 768, 769px to 1440px and above. You did well in styling the desktop view like the Figma design but the contents still need to be centered or aligned properly as close as possible to the Figma design.
- For screen sizes between the `mobile view - 375px` and `desktop view - 768px` should continue to display like a mobile view. This is were you have things to do to improve your project. 

Your project at a screen size of 686px 👇🏾 - this should still display like the mobile view. 
image

Your project at a screen size of `767px` 👇🏾 This should still display like the mobile view. 
image

Your project at screen-size of 768px 👇🏾 - Desktop view begins.
image

It should display like the Figma Desktop template. All its content should be properly aligned.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @ehmaddd, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ehmaddd) [reply index.html:43]: All such overflows were fixed in the newer version of stylesheet by introducing some more break points in between

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @ehmaddd, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**